src/cloud-api-adaptor/ibmcloud-powervs/image/prereq.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 GO_VERSION="1.21.12"
-RUST_VERSION="1.72.0"
```

Comment:
May be you can add a FIXME to pickup these values from versions.yaml as a future PR

Reply:
Added FIXME string, so that we can get all versions from versions.yaml.